### ISSUE TITLE
Tab key support is broken

### ISSUE THREAD

**Issue by waldyrious** [browserpad/browserpad#71]
Support for typing tab characters in the textarea was introduced all the way back in d2549d0efdd9efa05f76d98cd7e8163c29194a99, but it stopped working along the way, as @SenadAlagic pointed out in #70.

I've bisected the faulty change to commit 2e55678411bf267638240960732f796a7682dc61, which implemented the auto-save functionality. We need to restore the tab key functionality without messing with the auto-save.

**Comment by SenadAlagic** [comment]
I will definitely look into this! I'm glad you could pinpoint the exact commit where it happened. Cheers!

**Comment by SenadAlagic** [comment]
Hello! I looked into making "tab" indent without using the `onkeypress` function and I don't think it's doable, at least not in a simple way. Every Google search shows `onkeydown` being used (not even `onkeyup `which, also, doesn't work for some reason). `onkeypress` seems to only work with keys that produce characters. I implemented it with `onkeydown` but with a workaround so I check if the key has been let go or not, if the user keeps holding - no more than one indentation should happen. 
Let me know if that's something you like.
